src/views/graph-settings.js

```diff
@@ -162,14 +168,14 @@ export class GraphSettingsView extends ItemView {
   }
 
   /**
-   * Creates the UI to configure a given graph builder.
-   * @param {GraphBuilder} builder The builder to create the config UI for.
+   * Creates the UI to configure a selected graph builder.
    * @private
    */
-  createConfigUI_(builder) {
-    const config = builder.getConfig();
+  createConfigUI_() {
+    const config = this.selectedBuilder_.getConfig();
```

Comment:
Change this back so we are manually passing the builder. I want to rely on state as little as possible.